src/schemas/generator/v1/testOptions.ts

```diff
@@ -68,4 +69,5 @@ export const TestOptionsSchema = z.object({
   loadProfile: LoadProfileExecutorOptionsSchema,
   thinkTime: ThinkTimeSchema,
   thresholds: z.array(ThresholdSchema).default([]),
+  loadZones: LoadZoneSchema.default({ distribution: 'even', loadZones: [] }),
```

Comment:
I suggest nesting it under the `cloud` property, because cloud options have no effect in local tests and we may need to support more of them soon (namely, `projectID` and `name` - depends on how the run-in-cloud functionality is implemented):

```ts
cloud: {
  distribution: { /* or loadZones */ },
  projectID: 1234,
  name: 'my-script'
}
```